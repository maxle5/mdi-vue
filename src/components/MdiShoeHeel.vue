
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M3,18H11.7L17,14H18V18H20V14C20,14 21,12 21,10C21,8 20.5,6 20.5,6H18.5L18,7L10,14H8L3,16V18Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiShoeHeel",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
