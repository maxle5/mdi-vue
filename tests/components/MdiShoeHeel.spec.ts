
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiShoeHeel from "../../src/components/MdiShoeHeel.vue";

test("MdiShoeHeel snapshot", () => {
  const wrapper = mount(MdiShoeHeel, {});
  expect(wrapper.html()).toMatchSnapshot();
});
