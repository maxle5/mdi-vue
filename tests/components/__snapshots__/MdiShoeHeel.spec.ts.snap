// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiShoeHeel snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3,18H11.7L17,14H18V18H20V14C20,14 21,12 21,10C21,8 20.5,6 20.5,6H18.5L18,7L10,14H8L3,16V18Z" style="fill: currentColor;"></path>
</svg>"
`;
